agent/src/agent.cpp

```diff
@@ -103,13 +106,75 @@ class SoftwareContainerAgent
         return valid;
     }
 
+    ReturnCode readConfigElement(const json_t *element)
+    {
+        std::string wo;
```

Comment:
I personally think short variable names like this are more difficult to read. In this case it's not a problem as the code is short, but I don't see any benefit either. I mean, I want it to read e.g. "does write often equal one", not "does veee-ooh equal one". A matter of taste, just my two cents.